.vscode/launch.json

```diff
@@ -0,0 +1,48 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
May be we need to update gitignore to ignore this type of IDE files and folders.

Reply:
I commited that file accidentally. I removed it here: https://github.com/WordPress/theme-check/pull/458/commits/8d61a5995bed77cdcf42dad6259e7549bb0f61f4

I will update the .gitignore file in another PR.

Reply:
Submitted a PR for the gitignore: https://github.com/WordPress/theme-check/pull/460